modules/network-observability-multitenancy.adoc

```diff
@@ -22,3 +24,10 @@ $ oc adm policy add-cluster-role-to-user netobserv-reader user1
 +
 Now, the data is restricted to only allowed user namespaces. For example, a user that has access to a single namespace can see all the flows internal to this namespace, as well as flows going from and to this namespace.
 Project admins have access to the Administrator perspective in the {product-title} console to access the Network Flows Traffic page.
+
+* To enable multi-tenancy for developer perspective, run the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-role-to-user netobserv-metrics-reader <user_group_or_name> -n <your_namespace>
```

Comment:
in the above command we use an example user name `user1` and here we use a placeholder `<user_group_or_name>`, I guess we should align the two commands in a way or another

Reply:
Good point I will make them similar, user-replaced values.